template/cordova/lib/build.js

```diff
@@ -68,6 +70,40 @@ module.exports.run = function (argv) {
     }
 
     return parseAndValidateArgs(argv)
+    .then(function(buildopts) {
+        var csProjFilePath = shell.ls(path.join(ROOT, '*.csproj'))[0];
+        var WMAppManifestFilePath = path.join(ROOT, 'Properties', 'WMAppManifest.xml');
+        var AssemblyInfoFilePath = path.join(ROOT, 'Properties', 'AssemblyInfo.cs');
+        var configFilePath = path.join(ROOT, '..', '..', 'config.xml');
+
+        var configContents = removeBOM(fs.readFileSync(configFilePath, 'utf-8'));
+        var configXML =  new et.ElementTree(et.XML(configContents));
+        var defaultLocale = configXML.getroot().attrib['defaultlocale'] || 'en-US';
+
+        // Change locale on csproj file
+        var csProjContents = removeBOM(fs.readFileSync(csProjFilePath, 'utf-8'));
+        var csProjXML =  new et.ElementTree(et.XML(csProjContents));
+        var csProjDefaultLocale = csProjXML.find('./PropertyGroup/SupportedCultures');
+        csProjDefaultLocale.text = defaultLocale;
+        fs.writeFileSync(csProjFilePath, csProjXML.write({indent: 2}), 'utf-8');
+
+        // Change locale on WMAppManifest file
+        var WMAppManifestContents = removeBOM(fs.readFileSync(WMAppManifestFilePath, 'utf-8'));
+        var WMAppManifestXML =  new et.ElementTree(et.XML(WMAppManifestContents));
+        var WMAppManifestLanguages = WMAppManifestXML.find('./Languages/Language');
+        WMAppManifestLanguages.set('code', defaultLocale);
+        var WMAppManifestDefaultLanguage = WMAppManifestXML.find('./DefaultLanguage');
+        WMAppManifestDefaultLanguage.set('code', defaultLocale);
+        fs.writeFileSync(WMAppManifestFilePath, WMAppManifestXML.write({indent: 2}), 'utf-8');
+
+        // Change locale on AssemblyInfo file
+        var AssemblyInfoContents = removeBOM(fs.readFileSync(AssemblyInfoFilePath, 'utf-8'));
+        AssemblyInfoContents = AssemblyInfoContents.replace(/[\w\[: ]+NeutralResourcesLanguageAttribute\("[\w\-]+"\)]/,
```

Comment:
Changing the NeutralResourceLanguageAttribute does not look like the correct thing to do here.